### DISCUSSION TITLE
Decarbonize Software 2023

### DISCUSSION THREAD

**Discussion by NAMRATA-WOKE** [Green-Software-Foundation/discussions#54] [Events]
Registration for Decarbonize Software 2023 is officially open, along with a Call for Papers! The excitement is building faster than a software update, and to make sure we're setting you all up for success, we're opening up a discussion to gather your questions and insights. 

Do you have questions about our open-source projects or the insights shared in the [State of Green Software](https://stateof.greensoftware.foundation/) report, or insights about what we're doing that can be helpful to the community? This is the place to ask any questions or share your thoughts in advance of Decarbonize Software on November 16. 

We're anticipating a lot of questions this year. We'll do our best to get to each one during the event. If we're unable to in an effort to respect everyone's time, we'll make sure to address each question here!  

We hope you all register and take advantage of the Call for Papers. 

👉🏽 [Register today ](https://decarb.greensoftware.foundation/)
👉🏽 ~Submit an idea~  *CFP Deadline is September 25*

**Comment by adamj89** [comment]
Thank you to everyone who submitted a session for Decarbonize Software! Our call for speakers is now **closed** and we will soon be in touch with everyone.

**Reply by Dcvinci** [reply]
Hi Adam, I just aknoledged the presence of a deadline for the submission. Would it still be possible in any way to participate as a speaker? I am part of GSF standard Working group  and my team in NTT DATA is working on a very interesting Project centered around the calculation of software CO2e and SCI of a large bank.

**Reply by adamj89** [reply]
Hi @Dcvinci thanks, I got your talk in time. We will make a decision by early next week. We've had 10x the number of talks submitted that we have space for in the one event, so we will be creating other opportunities to hear all the amazing content that has been submitted.

**Reply by NAMRATA-WOKE** [reply]
@Dcvinci  We're so happy with all the contributions made by NTT Data and I'd like to figure out how and when we can best showcase some of the work you've done and are doing. Can you find a time when you, @adamj89 and I can sync up? https://calendly.com/d/4fv-3sy-3s8/25-mins-with-adam-namrata

**Reply by danuw** [reply]
@adamj89 / @NAMRATA-WOKE, when I submitted I realised there were some new fields I had not planned for even checking again the form a couple of days before the deadline. So I did not provide speakers details yet but can provide later. Could you let me know if needed please?

**Reply by NAMRATA-WOKE** [reply]
Thanks for informing us @danu. Yes, we’ll let you know if we need more information from you!
